### DISCUSSION TITLE
Video Encoding

### DISCUSSION THREAD

**Discussion by mukhtharcm** [supabase/supabase#2178] [Feature Requests]
Do you guys have any idea on How to encode a video that is uploaded to supabase storage?
Like I'm building an on-demand video streaming platform?

**Comment by silentworks** [comment]
@mukhtharcm you would need to do this with a third party video encoding service like transloadit, mux or cloudflare.

https://transloadit.com/services/video-encoding/
https://connect.mux.com/video-encoding
https://www.cloudflare.com/en-gb/products/cloudflare-stream/
